pyrene/src/components/Form/Form.tsx

```diff
@@ -0,0 +1,258 @@
+/* eslint-disable react/static-property-placement */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-floating-promises */
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import React from 'react';
+import { ValidationError, ObjectSchema } from 'yup';
+import { MultiselectOption, Filters, Options } from '../Filter/types';
+
+type ValuesOf<T> = T[keyof T];
+type KeysOf<T> = keyof T;
+
+type FormValues = Filters;
+type ToucheValues = {
```

Comment:
I would rename this type to `TouchedValues`

Reply:
> I would rename this type to TouchedValues
Thanks. Nice catch !